### DISCUSSION TITLE
Kibana Map visualization

### DISCUSSION THREAD

**Discussion by presianbg** [Security-Onion-Solutions/securityonion#3671] [Unsupported Versions]
Hi,

I'm trying to create  MAP visualizations for a custom dashboard in Kibana, but I'm not able to make it work so far at all.
Any ideas or guides will be much appreciated:
 
![image](https://user-images.githubusercontent.com/2163486/112962344-f4795100-914e-11eb-8806-ed8afaf3de14.png)

Cheers,
Presian

**Comment by weslambert** [comment]
This is really more of an Elastic question, but to answer your question in short, you'll need to use the geoip fields, which means using records with valid source/destination IPs and the associate geo info.

**Reply by presianbg** [reply]
Sorry if it was not appropriate to ask this here. I highly appreciate your time and effort to help us.

I completely agree with you that is not specific SO question, but since this is a `community` discussion forum... maybe someone from the community could help?

In the alert documents we have:
![image](https://user-images.githubusercontent.com/2163486/113298635-b2920b80-9304-11eb-94c5-59ca7c321f69.png)
 
 But looks like those fields are not in the required format, because I can't select neither of them in the MAP creation menu:
 
![image](https://user-images.githubusercontent.com/2163486/113298775-d81f1500-9304-11eb-9b76-7750e8f056f9.png)

So my wild guess is that I have to do some kind of mapping ?


 Cheers,
 Presian

**Reply by presianbg** [reply]
Looks like the difficulties I have creating a map visualization are related to #3152
